### PR TITLE
Feature/297 motion detail page

### LINKED ISSUE #297: Bug: Motion detail page loading after a minute
it takes very long time to load this page because we are using an external api from polkascan.io which is defect. and also it wouldn’t work on litentry network because polkascan doesn’t support that. maybe we can get all data required from chain.

- [x] See if we can remove the dependency to the polkascan.io for loading this page.

### PR DESCRIPTION
**Pull request by ahoseinian**
Fixes #297 
- Remove polkascan api and use the info on chain
- Don't show polkascan webview for litentry chain ( polkascan doesn't support that )
- Some jsx refactoring

### CODE REVIEW

File: src/presentational/AccountInfoInlineTeaser.tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ function AccountInfoInlineTeaser({display, judgements}: PropTypes) {
 const styles = StyleSheet.create({
   text: {fontFamily: monofontFamily, fontWeight: 'bold'},
   container: {
+    flex: 1,
```
Comment (silva-fj) [inline src/presentational/AccountInfoInlineTeaser.tsx:30]: Can you please check the screens where we use this component that is not breaking with this change? I remember adding `flex: 1` here was breaking the UI somewhere. (Also check the usage of the AddressInlineTeaser)

File: src/screen/MotionDetailScreen.tsx [hunk 2]
```diff
@@ -51,19 +51,35 @@ export function MotionDetailScreen(props: PropTypes) {
 
   const {data: motion} = useMotionDetail(params);
 
+  const {data} = useCouncilMembers();
+  const membersCount = data?.members.length ?? 0;
+  const {isCloseable, isVoteable, hasFailed, hasPassed} = useVotingStatus(motion?.votes, membersCount, 'council');
```
Comment (silva-fj) [inline src/screen/MotionDetailScreen.tsx]: How about getting the status from this hook and remove all the logic down below?

```suggestion
  const {status} = useVotingStatus(motion?.votes, membersCount, 'council');
```

File: src/screen/MotionDetailScreen.tsx [hunk 3]
```diff
@@ -51,19 +51,35 @@ export function MotionDetailScreen(props: PropTypes) {
...
+    ? 'Closed'
+    : hasPassed
+    ? 'Passed'
+    : 'Open';
+
   if (!motion) {
     return <LoadingView />;
   }
 
+  const proposer = motion.votes?.ayes[0];
```
Comment (silva-fj) [inline src/screen/MotionDetailScreen.tsx:62]: Is the proposer always the first element?

File: src/presentational/AccountInfoInlineTeaser.tsx (hunk 1 above)
Comment (ahoseinian) [reply src/presentational/AccountInfoInlineTeaser.tsx:30]: I checked everything seems to be fine now, not sure were was the problem before.

File: src/screen/MotionDetailScreen.tsx (hunk 3 above)
Comment (ahoseinian) [reply src/screen/MotionDetailScreen.tsx:62]: TBH i am not sure! in polkadot it doesn't show the proposer on motion page so I didn't have example code to double check. however, from what I checked. when somebody makes a new motion he/she will be automatically the first aye vote. that's where this came from

File: src/screen/MotionDetailScreen.tsx (hunk 2 above)
Comment (ahoseinian) [reply src/screen/MotionDetailScreen.tsx]: 8e333d1

File: src/screen/MotionDetailScreen.tsx (hunk 3 above)
Comment (silva-fj) [reply src/screen/MotionDetailScreen.tsx:62]: I wonder if they don't show it for a reason.. Let's create an issue to revisit this so we don't forget

**Review by silva-fj — APPROVED** [review]

File: src/screen/MotionDetailScreen.tsx (hunk 3 above)
Comment (ahoseinian) [reply src/screen/MotionDetailScreen.tsx:62]: yes sounds good. https://github.com/litentry/litentry-app/issues/321

**Pull request merged**